README.md

```diff
@@ -59,10 +58,11 @@ docker-compose up
 
 We recommend using docker-compose in non-detached mode when building containers for first time. If you have previously built ModelPolisher  containers use `docker-compose up --detach`.
 
-On running these commands, databases will be restored in respective containers. After databases are successfully set up, use the following command to run ModelPolisher(in `ModelPolisher/` directory):
+On running these commands, databases will be restored in respective containers. After databases are successfully set up, use the following command (in `ModelPolisher/` directory) to run ModelPolisher:
 ```
-docker-compose run -v <path_directory_having_models>:/models/ java java -jar /ModelPolisher-noDB-1.7.jar --input=/models/<model_name> --output=/models/output/<output_name> --annotate-with-bigg=true --add-adb-annotations=true
+docker-compose run -v <path_directory_having_models>:/models/ java java -jar /ModelPolisher-noDB-1.7.jar --input=/models/<model_name> --output=/models/output/<output_name> --annotate-with-bigg=true --add-adb-annotations=true --output-combine=true 
```

Comment:
Add the various output options should be documented in the README. Here only the options are added, but the list describing the various output arguments and there function is missing.

Reply:
Yes I have been working on updating the README and providing complete information from Wiki Pages, it will done by tommorow!